assets/pages/support-landing/supportLanding.jsx

```diff
@@ -3,30 +3,50 @@
 // ----- Imports ----- //
 
 import React from 'react';
+import { Provider } from 'react-redux';
 
 import SimpleHeader from 'components/headers/simpleHeader/simpleHeader';
 import Footer from 'components/footer/footer';
 import CirclesIntroduction from 'components/circlesIntroduction/circlesIntroduction';
 import ThreeSubscriptions from 'components/threeSubscriptions/threeSubscriptions';
 import WhySupport from 'components/whySupport/whySupport';
 import ReadyToSupport from 'components/readyToSupport/readyToSupport';
-import PatronsEvents from 'components/patronsEvents/patronsEvents';
 
+import { init as pageInit } from 'helpers/page/page';
 import { renderPage } from 'helpers/render';
 
+import type { CommonState } from 'helpers/page/page';
+
+import PatronsEvents from './components/patronsEvents';
+
+
+// ----- Types ----- //
+
+export type State = {
+  common: CommonState,
+  page: {||},
```

Comment:
Um, what? `{||}`

Reply:
That's the flow type for a Cylon.

Nah, that's an exact object type, it means this has to be an empty object. Because of [width subtyping](https://flow.org/en/docs/lang/width-subtyping/) flow would allow properties to get added to `page` without them needing to be defined in the type here.

I've done this for the moment because there's nothing in the page state tree, but there will be in a future PR. It means I'll remember to update the state when that happens because flow should shout at me if I don't.

Reply:
kthanks. Cylon 🙄